client/my-sites/plugins/plugins-browser/index.jsx

```diff
@@ -78,6 +79,11 @@ const PluginsBrowser = ( {
 
 	const hasBusinessPlan =
 		sitePlan && ( isBusiness( sitePlan ) || isEnterprise( sitePlan ) || isEcommerce( sitePlan ) );
+
+	const { data: paidPluginsRawList = [], isFetchingPaidPlugins } = useWPCOMPlugins( 'all' );
```

Comment:
```suggestion
	const { data: paidPluginsRawList = [], isFetchingPaidPlugins } = useWPCOMPlugins( 'featured' );
```

Let's pass the `featured` as type even if it's not supported by the be which currently returns everything but we will implement it as soon as we add more plugins 

cc: @gavande1

Reply:
Fixed in 033bd3f